Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -1149,15 +1146,21 @@ def fetch_incidents(client: Client,
 def main():
     os.environ['PAN_CREDENTIALS_DBFILE'] = os.path.join(gettempdir(), 'pancloud_credentials.json')
     params = demisto.params()
-    registration_id_and_url = params.get(REGISTRATION_ID_CONST).split('@')
+    registration_id_and_url = (params.get('credentials_id', {}).get('identifier') or params.get('reg_id')).split('@')
+    if not registration_id_and_url:
+        raise DemistoException('ID must be provided.')
```

Comment:
Wrong text in Exception - should be registration_id

Reply:
why? The display is  'ID' of reg_id